scripts/openshiftci-presubmit-integrationtests.sh

```diff
@@ -7,6 +7,7 @@ set -x
 
 export CI="openshift"
 export TIMEOUT="30m"
+export GINKGO_VERBOSE_MODE="-v"
```

Comment:
I don't see the verbose output in the integration tests running on the openshift cluster like before

Reply:
Locally it is disabled. To enable it locally see the doc changes in this pr.

Reply:
@mik-dass https://github.com/openshift/odo/pull/2145/files#r326962153